docs-api/frontity-packages/collections-packages/hooks/infinite-scroll-hooks.md

```diff
@@ -188,7 +133,7 @@ rendered, the next page of the archive is fetched. A list of the fetched pages i
 
 The `limit` prop in this case stands for the number of posts being shown, not the number of fetched pages. In the same way, the `fetchNext` shows the next post, and only fetches the next page of posts if needed.
 
-#### Parameters
+### Parameters
 
 It accepts an optional object with the following props:
 
```

Comment:
The description of the `limit` prop in the table below does seem to me to agree with the description of the `limit` prop in line 134 above.

Reply:
Same as https://github.com/frontity/api-reference/pull/26#discussion_r607139388